microsoft/azure-pipelines-tasks
# [AzureFileCopy] Pick up latest Azure PowerShell module on Hosted agen…

…t (#13067)

* update azure file copy to use latest available azure rm module on hosted agent environment

* update task version

* check for null for latestazurermmodulepath

* move psmoduleutility to common. afcv2 and afcv3

* refactor out get-endpointauthenticationscheme function

* use endpoint.type instead of authentication scheme to check whether or not to update env:psmodulepath

* update task version of tasks using common modules dependency

* remove endpoint utility as it is not needed

* load utility.ps1 before calling get-endpoint

* update minor version

Co-authored-by: Ashish Ranjan 💩 <asranja@microsoft.com>
